.eslintrc

```diff
@@ -7,66 +7,21 @@
       "error",
```

Comment:
This file needs to be updated to use the `@joshdb/eslint-config`, see other packages for reference.